markdown/events/2022/bytesize-survey_results.md

```diff
@@ -25,3 +25,87 @@ It is our hope that these talks / videos will build an archive of training mater
+
+[2:49](https://youtu.be/DYdPNPVa4wct=169)
+Let's start off with some community demographics. Firstly, as hopefully we already knew and hoped, we are very global community, which I love. The majority of users are based in the US and in the UK and certainly in Europe. That's fairly inevitable from our origins of the community and also mirrors the density of people working in the field in bioinformatics. But there was I think 36 different countries in the respondents list, which is fantastic. I'm sure that's up a lot since the last few years. That's really nice to see. We're increasingly spreading around the world. And let's see if we can push these numbers up and make that map go even more blue then next year.
+
+[3:37](https://youtu.be/DYdPNPVa4wct=217)
+The majority of people, we asked what your favourite primary language for reading and writing was. Most of you picked English, which is not a surprise. But there's quite a lot of people speaking other languages as well. We have a pretty terrible gender equality. I'm not sure it's just our fault. I think it's probably indicative of a wider issue, but that's definitely something that could, of course, be improved if anyone has any ideas. And yeah, there's a pretty wide range of people, lots of early stage researchers using Nextflow and lots of people well into their career as well. It's really nice to see these kinds of things and get a feel for who everyone is.
+
+[4:19](https://youtu.be/DYdPNPVa4wct=259)
+So I said there's lots of languages. You can see English is up there at the top, but there's a long, long tail. And interestingly, a lot of people bundled into that "other" category there. So, again, there's a nod to how diverse our community is already. And this is really useful, for example, if we want to prioritise any efforts to translate material. We know which language is the most important to our community.
+
+[4:45](https://youtu.be/DYdPNPVa4wct=285)
+Digging into a bit more of what it is that everybody does, very similar to last year, the majority of people who filled in the survey are classed themselves as bioinformaticians. Few other people with different categories, job categories is always difficult. You can look into the others category of people, some identity crisis issues going on there. But most people are bioinformaticians working with biological data. Lots of people within academia and research, but also a lot of people in biotech startups, especially that seems to be growing since last year, and pharma and clinical work. That's really interesting to see as Nextflow matures and gets more heavily adopted, it's branching out of academia a little bit into the wider community.
```

Comment:
```suggestion
Digging into a bit more of what it is that everybody does, very similar to last year, the majority of people who filled in the survey are classed themselves as bioinformaticians. Few other people with different categories, job categories is always difficult. You can look into the others category of people, some identity crisis issues going on there. But most people are bioinformaticians working with biological datasets. Lots of people within academia and research, but also a lot of people in biotech startups, especially that seems to be growing since last year, and pharma and clinical work. That's really interesting to see as Nextflow matures and gets more heavily adopted, it's branching out of academia a little bit into the wider community.
```